### PR TITLE
refactor: Deleted unnecessary files

### PR DESCRIPTION
**Pull request by brunoffranca**
## What ❔

- ~`.cargo/config.toml` is a remnant from when we used private repos. It should no longer be necessary.~
- ~`configs/` doesn't seem to be used at all.~
- `building-from-scratch` just contains an old docker file that was used for Coinbase. No longer used.

### CODE REVIEW

File: .cargo/config.toml [hunk 1]
```diff
@@ -1,2 +0,0 @@
-[net]
-git-fetch-with-cli = true
```
Comment (Deniallugo) [inline .cargo/config.toml:2]: If we one day want to use private dependencies. 
e.g. some critical fixes, we will need to install this value again

File: configs/.gitkeep
[the remark this answers is not in the document]
Comment (brunoffranca) [reply configs/.gitkeep:1]: Got it. Can't zk toolbox just create an empty folder when initializing an ecosystem if there's none?

File: .cargo/config.toml (hunk 1 above)
Comment (brunoffranca) [reply .cargo/config.toml:2]: To fix a critical issue we most likely just create a private copy of the repo. We can set the value again if the need arises anyway.

Comment (Deniallugo) [reply .cargo/config.toml:2]: I just don't want to fight with it in a rush.

**Pull request merged**